packages/compute-baseline/src/browser-compat-data/feature.ts

```diff
@@ -38,6 +38,15 @@ export class Feature {
     return `[Feature ${this.id}]`;
   }
 
+  /**
+   * Return the feature's tags as an array (wthether there are any tags or not).
+   */
+  get tags(): string[] {
+    return Array.isArray(this.data.__compat?.tags)
+      ? this.data.__compat?.tags
+      : [];
```

Comment:
This is probably less literal-minded:

```suggestion
    return this.data.__compat?.tags ?? [];
```